src/ngx_http_lua_module.c

```diff
@@ -972,20 +972,29 @@ ngx_http_lua_create_srv_conf(ngx_conf_t *cf)
      *      lscf->srv.ssl_cert_handler = NULL;
      *      lscf->srv.ssl_cert_src = { 0, NULL };
      *      lscf->srv.ssl_cert_src_key = NULL;
+     *      lscf->srv.ssl_cert_src_ref = 0;
      *
      *      lscf->srv.ssl_session_store_handler = NULL;
      *      lscf->srv.ssl_session_store_src = { 0, NULL };
      *      lscf->srv.ssl_session_store_src_key = NULL;
+     *      lscf->srv.ssl_session_store_src_ref = 0;
      *
      *      lscf->srv.ssl_session_fetch_handler = NULL;
      *      lscf->srv.ssl_session_fetch_src = { 0, NULL };
      *      lscf->srv.ssl_session_fetch_src_key = NULL;
+     *      lscf->srv.ssl_session_fetch_src_ref = 0;
      *
      *      lscf->balancer.handler = NULL;
      *      lscf->balancer.src = { 0, NULL };
      *      lscf->balancer.src_key = NULL;
+     *      lscf->balancer.src_ref = 0;
```

Comment:
A really minor styling detail: We generally only comment about the `set by ngx_pcalloc` if it is not initialized by this function. Since those fields are initialized to `LUA_REFNIL` below, no need to add these comments.

See http://lxr.nginx.org/source/src/http/ngx_http_core_module.c#3375 as an example.

Reply:
Correct and done, thanks!